front/app/components/ProjectAndFolderCards/ProjectAndFolderCardsInner.tsx

```diff
@@ -58,10 +58,12 @@ const ProjectAndFolderCardsInner = ({
   onChangeAreas,
   onChangeTab,
 }: Props) => {
+  const rootLevelOnly = search && search.length > 0 ? false : true;
```

Comment:
Just a super tiny and not that important detail. Maybe `search.length === 0`?

Reply:
Which part? The current way `const rootLevelOnly = search && search.length > 0 ? false : true;` makes sure search exists (not null/undefined) and then checks that it also has length > 0, so it's a string with at least one character

The alternate way would be `const rootLevelOnly = !search || search.length === 0 ? true : false;` so "if search doesn't exist, or search exists and it's length is zero, rootLevelOnly is true, otherwise it's false"

Reply:
> makes sure search exists (not null/undefined) and then checks that it also has length > 0, so it's a string with at least one character

So, if `search` is not blank, `rootLevelOnly` is `false`, right? It can be simplified to `!search` instead of `search && search.length > 0 ? false : true`. A few tests:

```js
const fun1 = (search) => !search;
const fun2 = (search) => (search && search.length > 0 ? false : true);
const compareFor = (search) => console.log(fun1(search) === fun2(search));

compareFor('');
compareFor(undefined)
compareFor(null)
compareFor('Minsk')
compareFor('0')
```

Originally, I meant the ternary operator could be replaced with a simpler statement if it returns true or false. E.g., `search.length === 0 ? true : false` is the same as `search.length === 0`, no ternary operator needed.

Reply:
Yeah, in this case I wrote it explicitly a bit more verbose, more for other developers to show that `search` exists, and if it exists, we're checking explicitly to see that it has a length greater than zero, meaning the user entered at least one character. It's a bit more code but I think it demonstrates what's going on slightly better than the shorter version

Reply:
Good for me as soon as you're aware of alternatives :slightly_smiling_face: